front-end/lambda-in/src/App.js

```diff
@@ -29,6 +30,7 @@ class App extends Component {
             {/* For testing purpose */}
             <Route path="/dev-signup" component={DevSignUp} />
             <Route path="/dev-login" component={DevLogin} />
+            <Route path="/Dev-Info-Edit" component={DevInfoEdit} />
           </Switch>
```

Comment:
Keep the path consistent with the other paths; the others are all lowercase

Reply:
makes sense! will do.